File: 1323-maximum-69-number/1323-maximum-69-number.py
class Solution:
    def maximum69Number (self, num: int) -> int:
        max_ = num
        num = str(num)
        for i in range(len(num)):
            cur = [c for c in num]
            cur[i] = '9'
            max_ = max(max_, int("".join(cur)))
        return max_

File: 1323-maximum-69-number/1323-maximum-69-number.kt
class Solution {
    fun maximum69Number (num: Int): Int {
        val str = StringBuilder(num.toString())
        val s = num.toString()
        for (i in s.indices) {
            if (s[i] == '6') {
                str[i] = '9'
                break
            }
        }
        return str.toString().toInt()
    }
}